src/openrct2/platform/linux.c

```diff
@@ -56,6 +56,8 @@ void platform_get_exe_path(utf8 *outPath, size_t outSize)
         log_fatal("failed to get process path");
 
     }
+#elif defined(__OpenBSD__)
+    strlcpy(exePath, "/usr/local/bin/", sizeof(exePath));
```

Comment:
Because we only care about our package system. If a user compiles it elsewhere, then it's up to them to edit this line.
OpenBSD doesn't have a sysctl like FreeBSD to get the path anyway (at least, not easily, and as far as we're concerned, without benefit).